esp/services/ws_codesign/ws_codesignService.cpp

```diff
@@ -90,3 +90,34 @@ bool Cws_codesignEx::onListUserIDs(IEspContext &context, IEspListUserIDsRequest
     resp.setUserIDs(userIds);
     return true;
 }
+
+bool Cws_codesignEx::onVerify(IEspContext &context, IEspVerifyRequest &req, IEspVerifyResponse &resp)
+{
+    const char* text = req.getText();
+    if (!text || !*text)
+    {
+        resp.setErrMsg("No text provided");
+        return false;
+    }
+    StringBuffer signer;
+    bool isValidSig = false;
+
+    try
+    {
+        isValidSig = queryCodeSigner().verifySignature(text, signer);
+    }
+    catch (IException *e)
+    {
+        StringBuffer msg;
+        e->errorMessage(msg);
+        auto code = e->errorCode();
```

Comment:
trivial: since this is used in a printf, it would be safer to use the explicit type i.e. unsigned.